src/mos.h

```diff
@@ -193,6 +194,14 @@ UINT8	fat_EOF(FIL * fp);
 #define HELP_TYPE			"Display the contents of a file on the screen\r\n"
 #define HELP_TYPE_ARGS		"<filename>"
 
+#define HELP_KEY			"Store a command in one of 12 hotkey slots assigned to F1-F12\r\n\r\n" \

```

Comment:
super-tiny gotcha in this file is that the `help` command response contents (`help help`) currently does not compose the list of commands, so you should explicitly add in `KEY` into `HELP_HELP`